### PR TITLE
fix: mail template stylesheet fix

### LINKED ISSUE #3875: Mail Stylesheet Missing
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[`mailer.html.erb`](https://github.com/CircuitVerse/CircuitVerse/blob/master/app/views/layouts/mailer.html.erb) required `mailer.css`. But it seems to be that somehow the stylesheet has been deleted.

**As a result, sidekiq can't find css in** 
- development environment
- production environment

**Screenshot of a welcome mail**
The mail doesn't have any design.

**Expected Behaviour**

### Steps To Reproduce

Initiate any email anyhow [By creating grp, assignment or register new account]

### Are you working on this issue?

Yes

### PR DESCRIPTION
**Pull request by tanmoysrt**
Fixes #3875 

#### Describe the changes you have made in this PR -
- Updated stylesheet_link of `layouts/mailer.html.erb`
- Recover mailer.scss from a old version of this repository

### Screenshot of Current Email Templates after changes -

**Welcome Mail**

**New Group Creation Mail**

**Add member to group Mail**

**Assignment Mail**

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/views/layouts/mailer.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" content="width=device-width, initial-scale=1.0">
     <title></title>
     <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
-    <%= stylesheet_link_tag "/css/mailer.css" %>
+    <%= stylesheet_link_tag :mailer %>
```
Comment (tachyons) [inline app/views/layouts/mailer.html.erb:7]: External CSS won't work in most of email clients including Gmail

Comment (tanmoysrt) [reply app/views/layouts/mailer.html.erb:7]: oh okk will fix !

Comment (tanmoysrt) [reply app/views/layouts/mailer.html.erb:7]: @tachyons can we use `premailer-rails` gem for injecting the CSS ?

Comment (tanmoysrt) [reply app/views/layouts/mailer.html.erb:7]: @tachyons added mailer.css to asset precompile
and premier already initialized, so the css's already converted to inline CSS before sending

By inspecting the source details of mail from MailCatcher we can see the stylesheet link converted to inline CSS by premailer
image

**Review by smritigarg — APPROVED** [review]
Thanks for the change @tanmoysrt Looks good to me

**Pull request merged**